src/Gui/TreeItemMode.h

```diff
@@ -1,5 +1,5 @@
 /***************************************************************************
- *   Copyright (c) 2004 J�rgen Riegel <juergen.riegel@web.de>              *
+ *   Copyright (c) 2004 Jürgen Riegel <juergen.riegel@web.de>              *
```

Comment:
How is it invalid? + the asterisk at the end of the line needs to be fixed as well

Reply:
See how second letter in Juergen's first name is '?'.  I don't know preciseely what is wrong with it, but throws build warnings, and also shows '?' in my IDE.

Reply:
I don't see it in vscode or on github.

Reply:
![Screenshot_20221130_065512](https://user-images.githubusercontent.com/4140247/204790055-aede98fc-ea87-4919-ae65-68b45c44cd10.png)